Gemfile

```diff
@@ -77,11 +77,6 @@ group :test do
   gem 'webdrivers'
 end
 
-group :linux, :production do
-  # Does not build on Mountain Lion nor is it needed on OS X
-  gem 'therubyracer'
```

Comment:
This was difficult to build on Apple's newer M1 chips, but I also didn't see it actually being used anywhere. If this is needed for something, please revert this commit.

Reply:
It is needed, I couldn't tell you _exactly_ why, but I know from experience that the site doesn't work without it :-/

Reply:
Please revert the commit and just comment it out in your dev environment.

Reply:
I reverted this change [here](https://github.com/bellroy/predictionbook/pull/203/commits/45c5f9a67ab4cf86337ecdb90f9ac003ae172080).